Controller/PageAdminController.php

```diff
@@ -94,7 +94,9 @@ public function treeAction()
 
         $this->get('twig')->getExtension('form')->renderer->setTheme($formView, $this->admin->getFilterTheme());
 
-        return $this->render('SonataPageBundle:PageAdmin:tree.html.twig', array(
+        $templates = $this->retrieveAdminTemplates();
+
+        return $this->render($templates['tree'], array(
```

Comment:
seem logic i'll work on today, the first problem was related with PHP 5.3 compatibility.
Is the `retrieveAdminTemplates()` function in its place ?